persistence/src/main/scala/hmda/persistence/institutions/InstitutionPersistence.scala

```diff
@@ -73,7 +78,7 @@ class InstitutionPersistence extends PersistentActor with ActorLogging {
       }
 
     case GetInstitutionById(institutionId) =>
-      val institution = state.institutions.find(x => x.id == institutionId).getOrElse(Institution())
+      val institution = state.institutions.find(x => x.id.toString == institutionId)
```

Comment:
Note: our choice of `Int` for the ID is not set in stone (see point 3.i [here](https://github.com/cfpb/hmda-platform/pull/452#issuecomment-236234651)), so feel free to keep using `String` if it makes your changes easier, or seems better for some other reason.  I realize you'll have to change some code either way, though.